### DISCUSSION TITLE
Theme extend not working - Tailwind with Next.js

### DISCUSSION THREAD

**Discussion by danilo-leal** [tailwindlabs/tailwindcss#3018] [Help] [answered]
Hey, y'all.

I have a lil Tailwind project set up with Next. I've just extended the theme with fontSize but it doesn't seem to make an effect. Whenever I add the `tiny` class, the text goes back to text-base for some reason. Does anybody know what's up? I'm not seeing any issue when compiling whatsoever. Haven't got a clue. 

Colors extensions work though. Weird huh?

```
  theme: {
    extend: {
      fontSize: {
        tiny: '9px',
      },
    },
  },
```

**Comment by simonswiss** [comment, marked as answer]
Hey! 👋

Be sure to use the `text-tiny` class and not `tiny` in your HTML.

If you've done that, and it's not working, we'll need a repo to look at as it's impossible to troubleshoot without more details.

What you're describing should work - here's a Tailwind Play example that adds a `tiny` font size in the config, literally copied from your example above:

https://play.tailwindcss.com/nBt1qPCMTv

**Reply by danilo-leal** [reply]
Dumb me! Thanks dude, worked fine now :)

**Reply by simonswiss** [reply]
Haha no worries, glad it's sorted 👍

**Reply by garrett-thompson** [reply]
Wow, this bit me too. Thanks @simonswiss

**Reply by simonswiss** [reply]
Hah no problem - glad to help many folks with this 🎉
